controller/test-files/work_item_type/list/ok_agile.res.payload.golden.json

```diff
@@ -355,19 +353,11 @@
             }
           },
           "business_value": {
-            "description": "The business value of the theme.",
-            "label": "Business value",
-            "required": false,
-            "type": {
-              "kind": "integer"
-            }
-          },
-          "effort": {
-            "description": "The effort needed to implement this theme.",
-            "label": "Effort",
+            "description": "Business value for this theme.",
+            "label": "Business Value",
             "required": false,
             "type": {
-              "kind": "float"
+              "kind": "string"
```

Comment:
@michaelkleinhenz @stevengutz what is the reasoning behind making the business value a string field?